### DISCUSSION TITLE
how is it used?

### DISCUSSION THREAD

**Discussion by apo-bozdag** [erayaydin/NVimArtisan#1] [Q&A] [answered]
I'm new to vim how can I use it :D

**Comment by erayaydin** [comment, marked as answer]
These are highly recommend for any new Vimmer

- Vim tutor
```
:Tutor
```
- In-built help docs
```
:h autocommand
:h vim.api
:h insert
:h <command>
```
- [Google it](https://google.com/)
Example format query, `nvim how to delete x lines`
- [NeoVim quick reference](https://neovim.io/doc/user/quickref.html)
- [Vim Adventures](https://vim-adventures.com/)
- [Vim Genius](http://www.vimgenius.com/)
